salesforce_api/gmail-to-salesforce-lead/Config.toml.example

```diff
@@ -0,0 +1,4 @@
+gmailAccessToken = ""
+openAIKey = ""
+salesforceBaseUrl = "https://instance-id.salesforce.com"
+salesforceAccessToken = ""
```

Comment:
Avoid committing config.toml

Reply:
This is an example Config.toml file for the user to get an idea of the structure. The actual Config.toml file is not committed.

Reply:
For other samples in this repo we followed the practice of adding config.toml structure to readme. So let's follow the same for all samples

Reply:
Fixed in [a718fba](https://github.com/ballerina-guides/integration-samples/pull/32/commits/a718fba876d7ed04e212bd0cd384c113178380cb)